blocks/init/src/blocks/custom/carousel-image/components/carousel-image-editor.js

```diff
@@ -7,13 +7,17 @@ export const CarouselImageEditor = (props) => {
       blockClass,
       media,
     },
+    actions: {
+      onChangeMedia,
+    },
   } = props;
 
   return (
     <div className={blockClass}>
       <ImageEditor
         blockClass={blockClass}
         media={(typeof media === 'undefined') || media}
```

Comment:
React handbook mentiones that logic shouldn't be written inline but extracted outside of the JSX block

https://infinum.com/handbook/books/frontend/javascript/react#passing-props

Reply:
This is how it's done on all our components if I'm not mistaken (37 results in 31 files).

While you have a valid point, refactoring it only here makes no sense and refactoring it in all files should surely be a separate PR.

Reply:
If it's in that many places I definitely agree, that this should be a separate PR 👍